templates/index.html

```diff
@@ -301,9 +301,9 @@ <h2>Open source operators in Python</h2>
 
 <section class="p-strip--light">
   <div class="row">
-    <div class="col-5 u-vertically-center">
-      <ul class="p-logo-section">
-        <li class="p-logo-section__item">
+    <div class="col-5 u-vertically-center p-logo-section">
+      <div class="p-logo-section_items">
```

Comment:
Could we update the alignment of the logos?
![image](https://user-images.githubusercontent.com/57550290/152308488-b1183049-0f93-4136-8625-adb4d275bb94.png)